packages/core/src/shared/fs/fs.ts

```diff
@@ -640,7 +640,7 @@ export class FileSystem {
      * @param path The file path for which to retrieve metadata.
      * @return The Uri about the file.
      */
-    #toUri(path: string | vscode.Uri): vscode.Uri {
+    toUri(path: string | vscode.Uri): vscode.Uri {
```

Comment:
this seems like a strange thing to expose as a public api on the "fs" interface. it's a minor convenience for the code local to this module. 

we have a `packages/core/src/shared/utilities/uriUtils.ts` module, should it live there (though still kind of questionable) ?

Reply:
uriUtils makes sense for this to live in, I'll move it there